index.md

```diff
@@ -59,11 +59,11 @@ A good position statement should be a few paragraphs long and should include:
 - Links to related supporting resources.
 - Any other topics you think the workshop should cover in order to be effective.
 - A focus on technical issues, not process or platform preference. We plan to talk about the what, not the how.
-- Position statements must be in English, preferably in HTML or plain-text format. You may include multiple topics, but we ask that each person submit only a single coherent position statement. The input provided at registration time (e.g., bio, goals, interests) will be published and linked to from this workshop page.
+- Position statements must be in English, preferably in HTML, markdown or plain text format. You may include multiple topics, but we ask that each person submit only a single coherent position statement. The input provided at registration time (e.g., bio, goals, interests) will be published and linked to from this workshop page.
 
 Submissions should be between 200 and 1000 words.
 
-Please email statements to the program committee at [permissions-ws-2022-organizers@chromium.org](mailto:permissions-ws-2022-organizers@chromium.org).
+Please submit statements to the program committee by adding your statement to the [position papers directory](https://github.com/w3c/permissions-ws-2022/tree/master/papers) and opening a [pull request](https://github.com/w3c/permissions-ws-2022/pulls). 
```

Comment:
I suggest we keep an email as an option bacause it works for everyone (while not all are well-versed in GH).

```suggestion
Please submit statements to the program committee by adding your statement to the [position papers directory](https://github.com/w3c/permissions-ws-2022/tree/master/papers) and opening a [pull request](https://github.com/w3c/permissions-ws-2022/pulls). Alternatively, you can email your statement to the program committee at [permissions-ws-2022-organizers@chromium.org](mailto:permissions-ws-2022-organizers@chromium.org).
```

Reply:
Thank you, I have updated the text in this commit:

https://github.com/w3c/permissions-ws-2022/commit/a43c1ec88b1f1f3db69b6e71c58f09f4077cd5a8